coder-sdk/env.go

```diff
@@ -81,50 +84,39 @@ type CreateEnvironmentRequest struct {
 
 // CreateEnvironment sends a request to create an environment.
 func (c Client) CreateEnvironment(ctx context.Context, orgID string, req CreateEnvironmentRequest) (*Environment, error) {
-	var env *Environment
-	err := c.requestBody(
-		ctx,
-		http.MethodPost, "/api/orgs/"+orgID+"/environments",
-		req,
-		env,
-	)
-	return env, err
+	var env Environment
+	if err := c.requestBody(ctx, http.MethodPost, "/api/orgs/"+orgID+"/environments", req, &env); err != nil {
```

Comment:
sprintf rather than string concatenation?

Reply:
There is a mix of both a bit everywhere in the repo. Here I just stopped the error without changing how the string is created.

Created #115 to track this.